client/src/main/java/se/fortnox/reactivewizard/client/Response.java

```diff
@@ -37,7 +38,29 @@ public HttpResponseStatus getStatus() {
      * @return the value of the header or null if the header is missing
      */
     public String getHeader(String header) {
-        return headers.get(header);
+        return httpClientResponse
+            .responseHeaders()
+            .get(header);
+    }
+
+    /**
+     * Fetching a cookie value
+     * @param cookieName the cookie name
+     * @return the value(s) of the cookie
```

Comment:
are the values the properties of the cookie? or can a cookie have multiple values? 

Should we have it case sensetive?

Reply:
It is the values that are returned. A cookie can have multiple values.
Cookie names are case sensitive